vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -796,7 +796,7 @@ bool AdsImpl::ShowAd(
   auto notification_info = std::make_unique<NotificationInfo>();
   notification_info->advertiser = ad_info.advertiser;
   notification_info->category = category;
-  notification_info->text = ad_info.notification_text;
+  notification_info->text = "[Ad] " + ad_info.notification_text;
```

Comment:
should this be localized?

Reply:
Issue already merged so I have asked the question and will raise a ticket if so, thanks